parser/index.js

```diff
@@ -12,7 +12,17 @@ if (args.length !== 2) {
     throw new Error('Usage: node index.js path_to_log_file path_to_csv_result');
 }
 
-mkdir('out')
+mkdirIfRequired('out')
     .then(() => readFile(args[0], 'utf8'))
-    .then(data => parse(data))
+    .then(data => parser.parse(data))
     .then(csvContent => writeFile('out/' + args[1], csvContent, 'utf8'));
+
+function mkdirIfRequired(path) {
```

Comment:
yes saw this issue too.
an alternative is to use mkdirp module (which does that for you)